packages/generator-langium/src/index.ts

```diff
@@ -133,6 +141,16 @@ export class LangiumGenerator extends Generator {
                 message: 'Include VSCode extension?',
                 default: true
             } as PromptQuestion<Answers>,
+            {
+                type: 'confirm',
+                name: 'includeExampleProject',
+                prefix: description(
+                    'You can generate an example project to play around with Langium (with a "Hello world" grammar, generator and validator).',
```

Comment:
The description of `extensionName` above is outdated:

> This tool generates a VS Code extension with a "Hello World" language to get started quickly.

Could you change it to this?

> This tool generates one or more npm packages to create your own language based on Langium.